data/serviceworker.mjs

```diff
@@ -64,24 +163,34 @@ export default {
 	onmessageerror: true,
 	onnotificationclick: true,
 	onnotificationclose: true,
-	onoffline: true,
-	ononline: true,
+	onpaymentrequest: true,
+	onperiodicsync: true,
 	onpush: true,
-	onpushsubscriptionchange: true,
```

Comment:
This does exists, https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope/pushsubscriptionchange_event

But chrome doesn't support https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope/pushsubscriptionchange_event#browser_compatibility